storage/store.go

```diff
@@ -1587,9 +1581,8 @@ func (s *Store) GetStatus() (*StoreStatus, error) {
 func (s *Store) computeReplicationStatus(now int64) (
 	leaderRangeCount, replicatedRangeCount, availableRangeCount int32) {
 	// Load the system config.
-	cfg, err := s.Gossip().GetSystemConfig()
-	if err != nil {
-		log.Error(err)
+	cfg := s.Gossip().GetSystemConfig()
+	if cfg == nil {
```

Comment:
we used to log Error. Now that the only possible error in this call is missing config, which is rather rare, I think Info is appropriate.